idd/Energy+.idd.in

```diff
@@ -52927,6 +53038,337 @@ AirConditioner:VariableRefrigerantFlow,
        \note Enter the time constant used to model the transition
        \note from cooling only mode to heat recovery mode
 
+AirConditioner:VariableRefrigerantFlow:FluidTemperatureControl,
+       \memo This object models the performance of the Variable Refrigerant Flow system 
+       \memo excluding the indoor units. 
+       \min-fields 21
+  A1 , \field Heat Pump Name
+       \required-field
+       \type alpha
+       \note Enter a unique name for this variable refrigerant flow heat pump
+  A2 , \field Availability Schedule Name
+       \type object-list
+       \object-list ScheduleNames
+       \note Enter the name of a schedule that defines the availability of the unit.
+       \note Schedule values of 0 denote the unit is off. All other values denote the unit is available.
+       \note If this field is left blank, the unit is available the entire simulation.
+  A3 , \field Zone Terminal Unit List Name
+       \required-field
+       \type object-list
+       \object-list ZoneTerminalUnitListNames
+       \note Enter the name of a ZoneTerminalUnitList. This list connects zone terminal units to this
+       \note heat pump.	   
+  A4 , \field Refrigerant Type
+       \required-field
+       \type object-list
+       \object-list FluidNames
+       \default R410A
```

Comment:
If there is a default value this input IS NOT a \required-field. Remove the \required-field tag.